### ISSUE TITLE
.NET site link leading into aspnet/Docs is incorrect

### ISSUE THREAD

**Issue by guardrex** [aspnet/Docs#5739]
The text prior to the link into the docs from the page at https://www.microsoft.com/net/learn/apps/web/get-started/windows is ...

> Now that you've got the basics, continue building your first ASP.NET app with Razor Pages.

... with a link for *Tutorial: Razor Pages*. The link URL is the RP landing page in the docs at ...

https://docs.microsoft.com/aspnet/core/mvc/razor-pages

If the text and link are correct, the link should go to the RP tutorial. The URL should be ...

https://docs.microsoft.com/aspnet/core/tutorials/razor-pages/razor-pages-start

cc/ @rowanmiller

**Comment by rowanmiller** [comment]
It seems like we have duplicate content here. Both of those URLs provide a tutorial to build a RP app. It seems like the second one is more complete, and has a nicer path to keep leading folks thru more concepts.

The one we currently link to has inline instructions for all OS's, whereas the second one if for Windows. Though on the marketing site end we could send to a different URL based on each OS easy enough.

Bottom line
- Agree we should change the marketing site - I'll take care of that
- Can we sort out the duplicate content issue... who should look at that?

**Comment by guardrex** [comment]
We don't classify the first one as a "tutorial," per se (if "tutorial" is defined as a set of steps we intend the reader to take to build an app from scratch). The first one is more of a show-'n-tell of what's in the template. The second one is meant to be more of a step-by-step, do-this/do-that walk-thru, where the reader takes action to build an app from scratch. Both of those are @Rick-Anderson's topics, so he'll be able to shed more light.

**Comment by guardrex** [comment]
Yeah, I guess broadly speaking, they are both "tutorials" by standard definitions. That second one tho has a lot of meat on the bone. It's really just the beginning of this experience: https://docs.microsoft.com/aspnet/core/tutorials/razor-pages/

**Comment by rowanmiller** [comment]
Totally agree that this is the right content to push folks to. The Get Started tutorial on the .NET marketing site as already dynamic based on OS, so I'm updating it to point to the right tutorial for the OS in question.

**Comment by guardrex** [comment]
If it's an internal issue, should I go ahead and close this?

**Comment by rowanmiller** [comment]
I think it's fine to track here, I'll have it live later today and we can close then

**Comment by rowanmiller** [comment]
Change is live, I think we can close this now.

**Comment by guardrex** [comment]
ty @rowanmiller ... closing now.
